App/Redux/ThreadsRedux.ts

```diff
@@ -28,6 +28,24 @@ const actions = {
   }),
   refreshThreadsError: createAction('REFRESH_THREADS_ERROR', resolve => {
     return (error: Error) => resolve({ error })
+  }),
+  addExternalInvite: createAction('ADD_EXTERNAL_THREAD_INVITE', resolve => {
```

Comment:
Just to be consistent, I'd name the actions that initiate the async sequence with 'request': `addExternalInviteRequest`, for example.